### PR TITLE
Add Test URLs for burmese somali bengali urdu indonesia nepali pashto turkce tamil swahili hausa, skip related content test for legacy maps

### PR DESCRIPTION
**Pull request by LilyL0u**
**Overall change:** 
Test URLs added for CPS MAPs and Legacy MAPs.

Related content test skipped for legacy MAPs because of unreliable data formats

**Code changes:**
Added URLs to config test sections

Added if clause with a log in the else, so related content test is skipped for legacy MAPs and logs that it was skipped

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/support/config/services.js [hunk 1]
```diff
@@ -426,7 +429,9 @@ const genServices = {
       mediaAssetPage: {
         paths: {
           live: [],
-          test: [],
+          test: [
```
Comment (karinathomasbbc) [inline cypress/support/config/services.js:432]: Do we need a Burmese CPS MAP too?

File: cypress/support/config/services.js [hunk 2]
```diff
@@ -2308,7 +2330,10 @@ const genServices = {
         paths: {
           live: [],
           test: [],
-          local: ['/turkce/haberler-dunya-50924340'],
+          local: [
```
Comment (karinathomasbbc) [inline cypress/support/config/services.js]: This is the PGL section - can we revert this change, can put back to `local: ['/turkce/haberler-dunya-50924340'],`

**Review by karinathomasbbc — CHANGES_REQUESTED** [review]

File: cypress/integration/pages/mediaAssetPage/tests.js [hunk 3]
```diff
@@ -77,17 +77,23 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
     it('should have href that matches assetURI for 1st related content link', () => {
       cy.request(`${Cypress.env('currentPath')}.json`).then(({ body }) => {
         const numRelatedContentGroups = body.relatedContent.groups.length;
-
-        if (numRelatedContentGroups > 0) {
+        const assetUriString = body.metadata.locators.assetUri;
+        if (
+          numRelatedContentGroups > 0 &&
+          assetUriString.split('/').length <= 5
```
Comment (RichardPK) [inline cypress/integration/pages/mediaAssetPage/tests.js]: For understandability - could `assetUriString.split('/').length <= 5` be assigned to an 'is not legacy asset' variable (or indeed an 'is legacy asset' variable as I think is the pattern within the rest of the tests?) and the variable used in this condition instead? 

Currently the only way of knowing what this if () statement is capturing is via the cy.log statement in the else.

File: cypress/support/config/services.js (hunk 2 above)
Comment (RichardPK) [reply cypress/support/config/services.js]: This array is within the PGPs object - should we be changing it at all?

**Review by RichardPK — CHANGES_REQUESTED** [review]

File: cypress/integration/pages/mediaAssetPage/tests.js (hunk 3 above)
Comment (karinathomasbbc) [inline cypress/integration/pages/mediaAssetPage/tests.js]: Let's create a function at the top level which determines if URL is legacy asset:

```
const isLegacyAsset = url => {
  return url.split('/').length > 4;
}
```

The refer to it in this condition:

```
if (numRelatedContentGroups > 0 && !isLegacyAsset(assetUri)) {
```

Comment (RichardPK) [reply cypress/integration/pages/mediaAssetPage/tests.js]: Karina has outlined the explicit solution to this here https://github.com/bbc/simorgh/pull/5768#pullrequestreview-369475560

File: cypress/support/config/services.js (hunk 2 above)
Comment (LilyL0u) [reply cypress/support/config/services.js]: Fixed

File: cypress/integration/pages/mediaAssetPage/tests.js (hunk 3 above)
Comment (LilyL0u) [reply cypress/integration/pages/mediaAssetPage/tests.js]: Changed, different method. Didn't see comment before working on it

**Review by karinathomasbbc — APPROVED** [review]

File: cypress/integration/pages/mediaAssetPage/tests.js [hunk 4]
```diff
@@ -78,16 +78,27 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
...
-          const assetURI =
-            body.relatedContent.groups[0].promos[0].locators.assetUri;
-          cy.get('li[class^="StoryPromoLi"] > div[class^="StoryPromoWrapper"]')
-            .find('h3')
-            .within(() => {
-              cy.get('a')
-                .should('have.attr', 'href')
-                .and('include', assetURI);
-            });
+        const requestPath = Cypress.env('currentPath');
```
Comment (ryanmccombe) [inline cypress/integration/pages/mediaAssetPage/tests.js:81]: This looks good :)

as a side note, you can `return` early from functions - this can avoid the need to have deeply nested conditionals-within-conditionals, and makes the code a bit easier to follow

eg:

```js
if (isLegacyAsset) {
  return cy.log('Test skipped because legacy MAP')
}

if (numRelatedContentGroups === 0) {
  return cy.log('Test skipped because no related content')
}

// We will only get this far if none of the above returns were hit
const assetURI = body.relatedContent.groups[0].promos[0].locators.assetUri;

// ...
```

**Review by ryanmccombe — APPROVED** [review]

File: cypress/integration/pages/mediaAssetPage/tests.js (hunk 4 above)
Comment (RichardPK) [reply cypress/integration/pages/mediaAssetPage/tests.js:81]: Ah nice. Yea this would be a good solution to eliminate those else(s) {}

**Review by RichardPK — APPROVED** [review]
🚢
